src/pages/home/report/ReportActionItem.js

```diff
@@ -586,7 +586,6 @@ function ReportActionItem(props) {
                             <ReportActionItemSingle
                                 action={parentReportAction}
                                 showHeader={!props.draftMessage}
-                                wrapperStyles={[styles.chatItem]}
```

Comment:
Why was this prop removed?

Reply:
Because I added it internally to **ReportActionItemSingle** by default. This component is being used in 2 places and the same style was passed twice.